Fix 12 displaying as 0 on some clocks

Oripy opened the pull request.

Oripy committed:
Fix 12 displaying as 0

meiamsome commented:
@waielal - this looks good to you as well, I presume? Just want a second view on this before merging

Oripy committed:
Fix character spacing in clock 3

This caused overlap of the space at the end of the clock text
« 12:34_ » with the first « 1 », making it disapear

waielal commented:
This looks good to me. 
But I just added a hour12() method to the sketch object :sweat_smile:
